lib/writeBufferComponentType.js

```diff
@@ -4,6 +4,14 @@ var WebGLConstants = Cesium.WebGLConstants;
 
 module.exports = writeBufferComponentType;
 
+/**
+ * Writes a value to a buffer as a particular component type.
+ *
+ * @param {Buffer} buffer The buffer to be written.
+ * @param {Number} componentType WebGLConstants value for component type.
+ * @param {Number} value The value to write.
+ * @param {Number} byteOffset The offset into the buffer to be written.
+ */
 function writeBufferComponentType(buffer, componentType, value, byteOffset) {
```

Comment:
Why does this function need `Component` in it?  Why not just `writeBufferComponent`?  This writes one component to a buffer, right?  The "type" doesn't need to be explicit in the name.